Algorithms/Search/BFS.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections.Generic;
+
+namespace BFS
+{
+    /// <summary>
+    /// Bread First Search implementation.
+    /// </summary>
+    public class BreadthFirst<T>
+    {
+        // Returns true if item exists.
+
+        public bool Find(IGraph<T> graph, T vertex)
+        {
+            return bfs(graph.ReferenceVertex, new HashSet<T>(), vertex);
+        }
+
+        // BFS implementation.
+
+        private bool bfs(IGraphVertex<T> referenceVertex,

```

Comment:
Codacy found an issue: [Rename method 'bfs' to match pascal case naming rules, consider using 'Bfs'.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6325512)